autogpt_platform/frontend/src/app/login/page.tsx

```diff
@@ -65,46 +91,13 @@ export default function LoginPage() {
     );
   }
 
-  async function handleSignInWithProvider(
-    provider: "google" | "github" | "discord",
-  ) {
-    const { data, error } = await supabase!.auth.signInWithOAuth({
-      provider: provider,
-      options: {
-        redirectTo:
-          process.env.AUTH_CALLBACK_URL ??
-          `http://localhost:3000/auth/callback`,
-      },
-    });
-
-    await api.createUser();
-
-    if (!error) {
-      setFeedback(null);
-      return;
-    }
-    setFeedback(error.message);
-  }
-
-  const onLogin = async (data: z.infer<typeof loginFormSchema>) => {
-    setIsLoading(true);
-    const error = await login(data);
-    setIsLoading(false);
-    if (error) {
-      setFeedback(error);
-      return;
-    }
-    setFeedback(null);
-  };
-
   return (
-    <div className="flex h-[80vh] items-center justify-center">
-      <div className="w-full max-w-md space-y-6 rounded-lg p-8 shadow-md">
-        <h1 className="text-lg font-medium">Log in to your Account </h1>
-        {/* <div className="mb-6 space-y-2">
+    <AuthCard>
+      <AuthHeader>Login to your account</AuthHeader>
+      {/* <div className="mb-6 space-y-2">
```

Comment:
Ditto

Reply:
I removed this since it's most likely going to look different anyway.